bedrock/firefox/templates/firefox/private-browsing.html

```diff
@@ -0,0 +1,169 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title %}{{ _('Firefox Web browser — Private Browsing with Tracking Protection — Mozilla') }}{% endblock %}
+{% block page_desc %}{{ _('Get more protection and the most privacy, only from Firefox. No other browser gives you so much control over your digital footprint.')}}{% endblock %}
+{% block body_id %}firefox-private-browsing{% endblock %}
+{% block body_class %}mozid{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_private_browsing' %}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {{ fxfamilynav('desktop', 'private-browsing') }}
+{% endblock %}
+
+{% block content %}
+
+<main role="main">
+  <section class="intro">
+    <div class="container">
+      <header>
+        <h1>{{ _('More protection. <br>The most privacy. <br>Only from Firefox.') }}</h1>
+        <div class="download-buttons">
+          <div class="firefox-latest-cta">
+            <a href="https://www.mozilla.org" class="button-flat-dark" data-highlight="privateWindow" role="button">{{ _('Try Private Browsing') }}</a>
+          </div>
+          <div class="non-firefox-cta">
+            <p>{{ _('To give the new Private Browsing a try, download Firefox.') }}</p>
+            <div class="desktop-download-button">
+              {{ download_firefox(small=True, icon=False) }}
+            </div>
+            <div class="android-download-button">
+              <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" data-button-type="download" data-interaction="download click" data-download-version="Firefox for Android">
+                {{ high_res_img('firefox/android/btn-google-play.png', {'alt': 'Get it on Google Play', 'width': '129', 'height': '45', 'l10n': True}) }}
+              </a>
+            </div>
+            <div class="ios-download-button">
+              <a href="{{ settings.APPLE_APPSTORE_FIREFOX_LINK }}&amp;ct=mozorg-ios_page-appstore-button">
+                <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}">
+              </a>
+            </div>
+          </div>
+          <div class="firefox-old-cta">
+            <p>{{ _('Your Firefox is not up to date. Download the latest version to use the new Private Browsing.') }}</p>
```

Comment:
Not sure if it matters, and probably to late for a string change but, seems a little strange that this string does include the "with Tracking Protection" part?